packager/live_packager_test.cc

```diff
@@ -1201,7 +1201,22 @@ INSTANTIATE_TEST_CASE_P(
         LivePackagerTestCase{
             5, "audio/en/init.mp4", LiveConfig::EncryptionScheme::SAMPLE_AES,
             LiveConfig::OutputFormat::TS, LiveConfig::TrackType::AUDIO,
-            "audio/en/%05d.m4s", false}));
+            "audio/en/%05d.m4s", false},
```

Comment:
Could we enable the `compare_samples` flag on these cases for a more thorough validation? (will likely need to update `ReadExpectedData()` to accommodate this)

Reply:
What should we compare against? Should we write out packaged samples and compare against those?

Reply:
I was thinking that we could compare against the source samples since packaging should not alter them

Reply:
Unfortunately that doesn't work with how the fragments are currently captured. From SIM, the media fragments don't capture the segment type boxes (eg `styp`). https://github.com/Pluto-tv/service-ingest-media/blob/feature/TRANS-5884-cmaf-service/internal/cmaf/mp4_reader.go#L106. So the fragments that are eventually passed into shaka only contain the `moof` and `mdat` boxes.

Shaka will add in the these box types as well as `sidx` box types for each fragment.

Reply:
Could you elaborate on what doesn't work?  The fact that Shaka can package the segments indicates that it is able to parse the samples.  So the the parsers used in this test should work.